src/templates/contributors/article-list.md

```diff
@@ -0,0 +1,26 @@
+project_path: /web/_project.yaml
+book_path: /web/resources/_book.yaml
+
+{# wf_auto_generated #}
+{# wf_updated_on: 1900-01-01 #}
+{# wf_published_on: 1900-01-01 #}
+
+# {{{ title }}} {: .page-title }
+
+{{#each articles}}
+
+## {{ title }}
```

Comment:
Let's make the title a link too. I think you should be able to do `## [{{ title }}]({{url}}))`

Reply:
DOne.